File: Domovoy/ClientApp/src/api/models/ServicePaymentInvoice.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Service } from './Service';

export type ServicePaymentInvoice = {
    id?: number;
    serviceId?: number;
    service?: Service;
    amount?: number;
    check?: string | null;
    isPaid?: boolean;
    repaymentTime?: string;
};

File: Domovoy/Models/ServicePaymentInvoice.cs
namespace Domovoy.Models;

public class ServicePaymentInvoice
{
    public int Id { get; set; }
    public int ServiceId { get; set; }
    public Service Service { get; set; }

    public decimal Amount { get; set; }
    public string Check { get; set; }
    public bool IsPaid { get; set; }
    // DeadLine
    public DateTime RepaymentTime { get; set; }
}